client/db/bolt/upgrades_test.go

```diff
@@ -200,13 +207,13 @@ func unpack(t *testing.T, db string) (string, func()) {
 		t.Fatal(err)
 	}
 	_, err = io.Copy(dbFile, r)
-
+	archive.Close()
+	dbFile.Close()
```

Comment:
We're done with both the archive and the extracted db here.  The `NewDB` call back in the test loads the DB by its file name.